components/ui/CompactFeature.vue

```diff
@@ -20,31 +20,45 @@ import { Component, Prop } from 'vue-property-decorator'
 
 @Component
 export default class extends Vue {
-  @Prop(String) icon: string | undefined
-  @Prop(String) title: string | undefined
-  @Prop(String) description: string | undefined
+  @Prop(String) icon!: string
+  @Prop(String) title!: string
+  @Prop(String) description!: string
 }
 </script>
 
 <style lang="scss" scoped>
 @import '~carbon-components/scss/globals/scss/typography';
 
+$feature-icon-size: 2.25rem;
+
 .compact-feature {
-  $feature-icon-size: 3.75rem;
+  background-color: $purple-70;
+  padding: $spacing-05;
+  padding-bottom: $spacing-09;
```

Comment:
Just curious: why are you adding a 3rem padding bottom here?

Reply:
I was attempting to match the designs, but I'll change it back to with consistent padding on the card.